### PR TITLE
URL Transfer

### PR DESCRIPTION
**Pull request by Quantalabs**
Cleans up the false urls and redirect to current repo after repo transfer.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ updates:
     schedule:
       interval: "daily"
     assignees:
-      - "Quantalabs"
+      - "epispot"
```
Comment (quantum9Innovation) [inline .github/dependabot.yml:8]: @epispot is not assignable; mention a team instead, like @epispot/maint

Comment (quantum9Innovation) [inline .github/dependabot.yml:8]: @epispot is not assignable; mention a team instead, like @epispot/maint

File: .github/workflows/claim.yml [hunk 2]
```diff
@@ -37,5 +37,5 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: 'Thank you @'+context.payload.sender.login+' for claiming this issue! 🎉 \n\nPlease reference this issue when you submit your Pull Request and make sure you follow the [contributing guidelines](https://github.com/Quantalabs/NCOVDashboard/blob/main/CONTRIBUTING.md).'
+              body: 'Thank you @'+context.payload.sender.login+' for claiming this issue! 🎉 \n\nPlease reference this issue when you submit your Pull Request and make sure you follow the [contributing guidelines](https://github.com/epispot/NCOVDashboard/blob/main/CONTRIBUTING.md).'
```
Comment (quantum9Innovation) [inline .github/workflows/claim.yml:40]: Bad URL

File: docs/model.md [hunk 3]
```diff
@@ -5,27 +5,30 @@ title: EpiJS Module - Model
...
-    *   [Parameters][2]
-    *   [get_data][3]
-        *   [Examples][11]
+
+*   [Model][2]
+    *   [Parameters][3]
+    *   [Examples][4]
+    *   [get_data][5]
+        *   [Parameters][6]
+        *   [Examples][7]
```
Comment (quantum9Innovation) [inline docs/model.md:14]: Why are we changing this?

File: docs/plots.md [hunk 4]
```diff
@@ -6,27 +6,30 @@
...
-*   `model` **[Array][5]** The output from the pre module function.
-*   `canvas` **[HTMLCanvasElement][6]** The canvas to show the output.
-*   `days` **[Number][7]** The amount of days to plot
-*   `colors` **[Array][5]** Custom colors for the graph, in the same order the compartments are. (optional, default `null`)
+*   `model` **[Array][8]** The output from the pre module function.
 *   `options` **[Object][8]** Optional. Custom configuration to pass into the options parameter for chart.js, defaults to:
 ```JSON
+*   `days` **[Number][10]** The amount of days to plot
+*   `colors` **[Array][8]** Custom colors for the graph, in the same order the compartments are. (optional, default `null`)
+*   `options` **[Object][11]** Optional. Custom configuration to pass into the options parameter for chart.js, defaults to:```JSON
```
Comment (quantum9Innovation) [inline docs/plots.md:32]: Again, unrelated changes

File: package.json [hunk 5]
```diff
@@ -12,17 +12,17 @@
...
-        "url": "git+https://github.com/Quantalabs/EpiJS.git"
+        "url": "git+https://github.com/epispot/EpiJS.git"
     },
     "keywords": [
         "disease modeling",
         "epidemiology",
         "disease modelling"
     ],
-    "author": "QLabs (@Quantalabs)",
+    "author": "QLabs (@epispot)",
```
Comment (quantum9Innovation) [inline package.json:22]:
```suggestion
    "author": "QLabs (@Quantalabs)",
```

**Review by quantum9Innovation — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (Quantalabs) [reply package.json:22]: Instead - 
```suggestion
    "author": "epispot (@epispot)",
```

File: .github/dependabot.yml (hunk 1 above)
Comment (Quantalabs) [reply .github/dependabot.yml]: Fixed

File: .github/workflows/claim.yml (hunk 2 above)
Comment (Quantalabs) [reply .github/workflows/claim.yml]: Fixed

File: docs/model.md (hunk 3 above)
Comment (Quantalabs) [reply docs/model.md:14]: When we regenerated the documentation for new URLs, this might have been changed.

File: docs/plots.md (hunk 4 above)
Comment (Quantalabs) [reply docs/plots.md]: Outdated.

**Review by quantum9Innovation — APPROVED** [review]
